src/test/java/com/microsoft/sqlserver/jdbc/connection/ConnectionTest.java

```diff
@@ -69,4 +71,20 @@ public void testInvalidConnectWithIPAddressPreference() throws SQLException {
         }
     }
 
+    @Test
+    @Tag(Constants.xAzureSQLDW)
+    @Tag(Constants.xAzureSQLDB)
+    @Tag(Constants.xAzureSQLMI)
+    @Tag(Constants.xUnix)
+    public void testConnectionWithIntegratedSecurityWhenAuthDLLIsNotProvided() throws SQLException {
+        SQLServerDataSource ds = new SQLServerDataSource();
+        ds.setURL(connectionString);
+        ds.setUser("");
+        ds.setPassword("");
+        ds.setIntegratedSecurity(true);
+
+        // Driver should use packaged auth DLL
+        try (Connection con = ds.getConnection()) {}
+    }

```

Comment:
how does this test know whether the DLL was packaged?

Reply:
We'll need to stop loading the DLL in our pipelines for our tests.